openedx/core/djangoapps/programs/tests/test_backpopulate_program_credentials.py

```diff
@@ -81,42 +79,38 @@ def test_handle(self, commit, mock_task, mock_get_programs):
     @ddt.data(
         [
             factories.Program(
-                organizations=[factories.Organization()],
-                course_codes=[
-                    factories.CourseCode(run_modes=[
-                        factories.RunMode(course_key=course_id),
+                courses=[
+                    factories.Course(course_runs=[
+                        factories.CourseRun(key=course_id),
                     ]),
                 ]
             ),
             factories.Program(
-                organizations=[factories.Organization()],
-                course_codes=[
-                    factories.CourseCode(run_modes=[
-                        factories.RunMode(course_key=alternate_course_id),
+                courses=[
+                    factories.Course(course_runs=[
+                        factories.CourseRun(key=alternate_course_id),
                     ]),
                 ]
             ),
         ],
         [
             factories.Program(
-                organizations=[factories.Organization()],
-                course_codes=[
-                    factories.CourseCode(run_modes=[
-                        factories.RunMode(course_key=course_id),
+                courses=[
+                    factories.Course(course_runs=[
+                        factories.CourseRun(key=course_id),
                     ]),
-                    factories.CourseCode(run_modes=[
-                        factories.RunMode(course_key=alternate_course_id),
+                    factories.Course(course_runs=[
+                        factories.CourseRun(key=alternate_course_id),
                     ]),
                 ]
             ),
         ],
         [
             factories.Program(
-                organizations=[factories.Organization()],
-                course_codes=[
-                    factories.CourseCode(run_modes=[
-                        factories.RunMode(course_key=course_id),
-                        factories.RunMode(course_key=alternate_course_id),
+                courses=[
+                    factories.Course(course_runs=[
+                        factories.CourseRun(key=course_id),
```

Comment:
No, the ids are used. The backpopulation command checks those run ids when finding candidates for certification.